### PR TITLE
For the validator web ui, support specifying a URL to load, via #url=<some url>.

### PR DESCRIPTION
**Pull request by powdercloud**

### CODE REVIEW

File: validator/webui/index.html [hunk 1]
```diff
@@ -474,7 +474,22 @@
...
           });
 
-          var minimumValidAmp =
+          // Interprets the parameters after the '#' in the URL.
+          var paramStrings = window.location.hash.substr(1).split('&');
+          var params = {};
+          for (var ii = 0; ii < paramStrings.length; ii++) {
+            var keyValue = paramStrings[ii].split('=');
+            if (keyValue[0].length > 0) {
+              params[keyValue[0]] = keyValue[1];
```
Comment (honeybadgerdontcare) [inline validator/webui/index.html]: We can't be sure there's a value in keyValue[1], but maybe you're okay with that just being undefined.

e.g. (webui.../#asdf)

File: validator/webui/index.html [hunk 2]
```diff
@@ -474,7 +474,22 @@
...
+            }
+          }
+          // If #url=<url> was provided, load it, otherwise, start with the
+          // minimum valid AMP document.
+          if (params.hasOwnProperty('url')) {
+            input.value = params['url']
```
Comment (honeybadgerdontcare) [inline validator/webui/index.html]: Given above (e.g. params['url'] may be undefined due to "webui.../#url"), should we do a sanity check here before either giving input the value or before firing the click?

File: validator/webui/index.html (hunk 1 above)
Comment (powdercloud) [inline validator/webui/index.html]: Yeah I think undefined is ok here for cases when there's no value provided.

File: validator/webui/index.html (hunk 2 above)
Comment (powdercloud) [inline validator/webui/index.html]: Thank you, I changed this so it checks the value, so now it'll only use it if it's a non-empty string.

File: validator/webui/index.html (hunk 1 above)
Comment (honeybadgerdontcare) [inline validator/webui/index.html]: SGTM

**Comment by honeybadgerdontcare** [comment]
LGTM

**Pull request merged**